package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "main": "lib/index.js",
   "engines": {
-    "node": ">= 6.9"
+    "node": ">= 8.9"
```

Comment:
This was a breaking change in https://github.com/babel/babel-loader/pull/822, but no one complained :shrug: 

Also, AVA doesn't work on Node.js 8 so I'm running a smaller single test.

Reply:
🤔

Reply:
`schema-utils`, introduced in that PR, uses some syntax that is not supported on Node.js 6.